rtc-video-quality/metrics_template.html

```diff
@@ -0,0 +1,441 @@
+  "ssim-v": "SSIM Quality (-1,1)",
+  "avg-psnr":   "PSNR Quality in decibels",
+  "avg-psnr-y": "PSNR Quality in decibels",
+  "avg-psnr-u": "PSNR Quality in decibels",
+  "avg-psnr-v": "PSNR Quality in decibels",
+  "glb-psnr":   "PSNR Quality in decibels",
+  "glb-psnr-y": "PSNR Quality in decibels",
+  "glb-psnr-u": "PSNR Quality in decibels",
+  "glb-psnr-v": "PSNR Quality in decibels",
+  "psnr-dmos": "PSNR DMOS",
+  "encode-time-utilization": "Time (ms)",
+  "actual-encode-time-ms": "Time (ms)",
+  "vmaf": "VMAF (0,100)"
+};
+var selected = 0;
+var imagestr = '';
+var bettertable=0;
+var chart=0;
+var better=0;
+var metricdata=0;
+var metricView=0;
+var column=1;
+var formatter=0;
+
+function changeColumn(col) {
+  column = col;
+  console.log(col)
+  draw_files();
+}
+
+function changeMetric(m) {
+  ftable=m
+  draw_files()
+}
+
+function setup_vis() {
+  chart = new google.visualization.ScatterChart(
+      document.getElementById("metricgraph"));
+
+  bettertable = new google.visualization.Table(
+      document.getElementById("bettertable"));
+
+  draw_files();
+  build_metrics_radio();
+}
+
+function build_metrics_radio() {
+  for (metric=1; metric < metrics.length; metric++) {
+    var rb = document.createElement('input');
+    var l = document.createElement('label');
+    rb.setAttribute('type','radio');
+    rb.setAttribute('name','metric');
+    rb.setAttribute('onClick', "changeColumn('"+metric.toString()+"')");
+    l.innerHTML = metrics[metric];
+    document.getElementById('metrics').appendChild(rb);
+    document.getElementById('metrics').appendChild(l);
+  }
+}
+
+function draw_files() {
+  var options = {'allowHtml': true, 'width': "100%", 'height': "50%"};
+  if (better != 0) delete better;
+
+  col=eval(ftable+'[column]')
+  better = new google.visualization.DataTable(col)
+
+  // Python Template code replaces the following line with a list of
+  // formatters.
+  if (ftable == 'filestable_dsnr')
+    formatter = new google.visualization.NumberFormat(
+      {fractionDigits: 4, suffix:" db"});
+  else
+    formatter = new google.visualization.NumberFormat(
+       {fractionDigits: 4, suffix:"%"});
+
+  //%%formatters%%//
+
+  bettertable.draw(better,options);
+  google.visualization.events.addListener(bettertable, 'select',
+                                          selectBetterHandler);
+  query_file()
+}
+
+function query_file() {
+  imagestr = better.getFormattedValue(selected, 0)
+  var metricjson = eval('(' + snrs[column][selected] + ')');
+  metricdata = new google.visualization.DataTable(metricjson, 0.6);
+  if( metricView != 0 ) delete metricView;
+  metricView = new google.visualization.DataView(metricdata);
+  chart.draw(metricView, {curveType:'function',
+      explorer: {},
+      chartArea:{left:chart_left, top:chart_top, width:chart_width,
+      height:chart_height-90},
+      hAxis:{title:"Datarate in kbps"},
+      vAxis:{title:metricToUnit[metrics[column]], format: '##.0', textPosition: 'in'},
```

Comment:
Let's add a 'default' unit so that if new metrics are added they won't break this.

Reply:
Updated in 3c985f9